Supporting the 52Fund (#21)
digital-preservation-is-people/digital-preservation-is-people.github.io: _posts/2020-01-26-support-the-52Fund.md

> unsatisfied with the lack of a choice of two candidates with significant
> organizational experience posed by the slate recently put forward for the
> office of SAA vice-president/president-elect … I feel strongly that SAA
> members need a choice of candidates who have deep, varied experience working
> for our profession within our national organization, and multi-faceted
> connections to the concerns of SAA’s leadership.”

## Why is this a problem?

By-laws aside, 52 signers decided to subvert an open-ballot, open to an entire
organization to have their say, and put their weight behind someone they
considered credible.

52 signers sought an end-around of the current Chair of the Nominating
Committee. Themselves [seeking to][saa-6] do the following:

> I think it will be critical to put a slate of candidates together that will
> have a strong portfolio of success in making room for historically
> underrepresented identities in leadership positions, who advocate for success
> of these communities and are willing to call out and address discriminatory
> practices within the profession and in spaces supposedly designed to nurture

> > and support emerging leaders and change agents.

Reviewer: Add an attribution to this quote, like who exactly said it/wrote it and the context

We know you (the 52 signers) felt the nominees didn't bring enough "varied"
experience, and felt they didn't bring multi-faceted connections to the
concerns of the SAA's leadership (whatever that means?!).

> But what spoke to you and said:

Reviewer: ```suggestion

```

> * This truly didn't exist in the nominee's backgrounds, within SAA, or outside?

Reviewer: ```suggestion
* The nominees didn't bring enough varied leadership experience from within and outside SAA
```

> * That spoke to you and said, this couldn't be learned with a year in a VP

Reviewer: ```suggestion
* The complexities of the role could not be learned in a year in a VP position with the mentor-ship of the current president
```

> position with the mentorship of the current president?

Reviewer: ```suggestion

```

> * That you had the answer to what leadership should and does look like?

Reviewer: ```suggestion
* Leadership should and does look a certain way that is not congruent with what the nominees offer.
```

> * That you could speak better to the organization's needs than the existing

Reviewer: ```suggestion
* A candidate not chosen by the elected nomination committee better serves SAA's needs
```

> nominating committee?

Reviewer: ```suggestion

```

## Leadership in our community

Leadership in our community -- ultimately, the entire GLAM sector is the
strength, empathy, and sympathy, of each of our members. It relies on
individuals of all backgrounds to push back, and upwards, and to call-out the
behaviors of those with leadership in their title. To call out the behaviors of
organizations that have leadership in their mandate and are failing to
demonstrate that. That is what we mean by leadership by nature. Processes and
procedures can often help us but this time they are failing a group of good
people.

The SAA community, and existing leadership (by virtue of _position_, or
_**nature**_, have spoken and are looking to support Lae’l Hughes-Watkins
[call][saa-7].

> They have setup a fund for underrepresented communities to access to have

Reviewer: ```suggestion
They have setup the *[#52Fund][saa-8]* to support QTBIPOC, BIPOC, LGBTQ+, and disabled memory workers' participation in official professional channels, like renewing SAA memberships and attending SAA's annual meeting.
```

> their voice on this situation as well and that is the *[#52Fund][saa-8]*.

Reviewer: ```suggestion

```

> Please read about the mission, and please consider the ways that you can

Reviewer: ```suggestion
Please read about the mission and consider the ways that you can
```

> donate or support the cause.

Reviewer: ```suggestion
participate, donate, or support the cause.
```

> More on this can be learned by following the hash-tag

Reviewer: ```suggestion
More on this issue can be learned by following the hash-tag
```

[#thatdarnedpetition][saa-9] on Twitter.

> *Disclaimer*: The author of this post is not a member of the SAA, their

Reviewer: ```suggestion
*Disclaimer*: Though the author of this post is not a member of the SAA, their
```
